IMDA-BTG/aiverify
# add try except case in try to identify case where exception may happe…

…n and needed to be caught to return the correct value.

# Pull Request Template

## Description

Fix where an exception is raised, and the error message is empty when returned.

## Motivation and Context

The error message set in the HSET when it happen is empty.
With this Fix, it can now return the correct response and the error message is now set.

## Type of Change



fix: A bug fix









## How to Test

Run a data classification pipeline file where it can deserialize but unable to verify the model type.

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x ]  I have tested the changes locally and verified that they work as expected.
- [x ]  I have followed the project's coding conventions and style guidelines.
- [x ]  I have performed a self-review of my own code.

## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]
